de.dlr.sc.virsat.project.test/src/de/dlr/sc/virsat/project/test/AProjectTestCase.java

```diff
@@ -49,8 +46,8 @@
 	protected static final int MAX_TEST_CASE_TIMEOUT_SECONDS = 30;
 	protected static final int MAX_TEST_CASE_WAIT_TIME_MILLI_SECONDS = 1000;
 	
-	@Rule
-	public TestRule globalTimeout = new DisableOnDebug(Timeout.seconds(MAX_TEST_CASE_TIMEOUT_SECONDS));
+	//@Rule
+	//public TestRule globalTimeout = new DisableOnDebug(Timeout.seconds(MAX_TEST_CASE_TIMEOUT_SECONDS));

```

Comment:
Do we want to disable this rule or was this just somehow for the release?

Reply:
And if we don't want to have it why not deleting it?

Reply:
Nice catch. We only disabled it temporarily for debugging, its supposd to be renabled,